scripts/ci-runner.Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM summerwind/actions-runner:v2.278.0-ubuntu-20.04
+
+USER root
+
+# Install system requirements
+RUN apt-get update && \
+    # Global requirements
+    DEBIAN_FRONTEND=noninteractive apt-get install --yes \
+    build-essential \
+    curl \
+    # If we don't need gcc, we should remove it.
+#    g++ \
+#    gcc \
```

Comment:
Let's go ahead and remove the commented out lines here, if we don't actually need them.  I suspect many of the dev packages were once needed to install the Python bindings for them, but most of those are now available as pre-compiled manylinux wheels.  (Note that `build-essential` pulls in many large packages though, so it would be great if someday we didn't need that anymore.)